__tests__/app.test.js

```diff
@@ -49,3 +49,50 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200: responds with an article object corresponding to the passed id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) => {
+        const {article} = body;
+        expect(article).toHaveProperty("author", "butter_bridge");
+        expect(article).toHaveProperty(
+          "title",
+          "Living in the shadow of a great man"
+        );
+        expect(article).toHaveProperty("article_id", 1);
```

Comment:
Great to assert that api/articles/1 gives you back an id of 1. The other properties I would be less specific about hardcoding, just assert that they exist. You might decide to swap out the data in future and you wouldn't be worried about the specific image URL necessarily :)